examples/authorization_details_mso_doc.json

```
@@ -2,15 +2,23 @@
   {
     "type": "openid_credential",
     "credential_configuration_id": "org.iso.18013.5.1.mDL",
-    "claims": {
-      "org.iso.18013.5.1": {
-        "given_name": {},
-        "family_name": {},
-        "birth_date": {}
+    "claims": [
+      {
+        "namespace": "org.iso.18013.5.1",
+        "claim_name": ["given_name"]
```

Based on the definition [here](https://github.com/openid/OpenID4VCI/pull/276/files#diff-1f424614b35a9899813079f1b1f6218631a2aedd993368ccb89bb81a9eda0289R2312-R2315)

Should all instances of `claim_name` have a value type of string not an array of strings?
```suggestion
        "claim_name": "given_name"
```